src/lib.rs

```diff
@@ -79,6 +80,11 @@ const INIT_REQUESTS: ctypes::c_ulong =
     sspi::ISC_REQ_SEQUENCE_DETECT | sspi::ISC_REQ_MANUAL_CRED_VALIDATION |
     sspi::ISC_REQ_ALLOCATE_MEMORY | sspi::ISC_REQ_STREAM | sspi::ISC_REQ_USE_SUPPLIED_CREDS;
 
+const SEC_APPLICATION_PROTOCOL_LIST_HEADER_SIZE: usize =
+    mem::size_of::<u32>() + mem::size_of::<ctypes::c_ushort>();
```

Comment:
It's actually 2 bytes too large, because `size_of::<SEC_APPLICATION_PROTOCOL_LIST>()` accounts for the padding that the struct would have if it didn't have a variable-length array at the end.

I could do `size_of::<SEC_APPLICATION_PROTOCOL_LIST>() - size_of::<ctypes::c_ushort>()`, but I figured this was more readable. I can add a comment about why `size_of::<SEC_APPLICATION_PROTOCOL_LIST>()` isn't sufficient though.